.config/dotnet-tools.json

```diff
@@ -9,7 +9,7 @@
       ]
```

Comment:
Would it be worth updating Paket while we're updating everything else? It looks like it's on an RC version. (I'm putting the comment here because I can only comment on changed sections)